jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/oracle/OracleSqlGenerationVisitor.java

```diff
@@ -496,53 +482,55 @@ public String visit(final SqlFunctionScalar function) throws AdapterException {
             break;
         }
 
-        final boolean isDirectlyInSelectList = (function.hasParent() && function.getParent().getType() == SqlNodeType.SELECT_LIST);
+        final boolean isDirectlyInSelectList = (function.hasParent()
+                && (function.getParent().getType() == SqlNodeType.SELECT_LIST));
         if (isDirectlyInSelectList && this.scalarFunctionsCast.contains(function.getFunction())) {
             // Cast to FLOAT because result set metadata has precision = 0, scale = 0
```

Comment:
Remove the confusing comment.

Reply:
This is code I took over. Will be replaced by `sql-statment-builder` anyway soon.